src/main/java/org/opensearch/security/rest/SecurityInfoAction.java

```diff
@@ -112,6 +112,9 @@ public void accept(RestChannel channel) throws Exception {
                     builder.field("principal", (String)threadContext.getTransient(ConfigConstants.OPENDISTRO_SECURITY_SSL_PRINCIPAL));
                     builder.field("peer_certificates", certs != null && certs.length > 0 ? certs.length + "" : "0");
                     builder.field("sso_logout_url", (String)threadContext.getTransient(ConfigConstants.SSO_LOGOUT_URL));
+                    builder.field("tenancy_enabled", evaluator.multitenancyEnabled());
```

Comment:
the name `tenancy_enabled` is slightly misleading. Can we have `multi_tenancy_enabled` instead?

Reply:
Ok will make this change.